.gitignore

```diff
@@ -7,7 +7,7 @@
 .venv
 
 # build
-/build*
+**/build*
```

Comment:
Thanks. Please move it to after the `/twister-out*` line as well.

Reply:
You were faster than me. It is done now here: c19fab3f2d26d1ddb0a274fe7358145abbcdfdee